### PR TITLE
Add --iteration flag to vgg16_benchmark

### PR DESCRIPTION
**Pull request by shinh**
so we will get more stable number. Also, now it up to three
digits after the decimal point.

### CODE REVIEW

File: benchmark/vgg16_benchmark.cpp [hunk 1]
```diff
@@ -50,14 +51,18 @@ int main(int argc, char** argv) {
...
-                   .count()
-              << std::endl;
+        // Run inference
+        model.run();
+
+        auto end = clock::now();
+        usecs.push_back(std::chrono::duration_cast<std::chrono::microseconds>(end - start));
+    }
+
+    auto total_usec = std::accumulate(usecs.begin(), usecs.end(), std::chrono::microseconds());
```
Comment (okdshin) [inline benchmark/vgg16_benchmark.cpp]: I wonder that `std::chrono::microseconds()` is specified always to be zero?

Comment (shinh) [reply benchmark/vgg16_benchmark.cpp]: I don't know, but

* https://en.cppreference.com/w/cpp/chrono/duration/duration says the default constructor is the default one and
* https://en.cppreference.com/w/cpp/chrono/duration says `std::chrono::microseconds` is `duration</*signed integer type of at least 55 bits*/, std::micro>`

so I think it's safe to assume the default value is zero. That said, I think it's good to use `zero` to make the intention clearer. I changed the default constructor to `zero`.

Comment (okdshin) [reply benchmark/vgg16_benchmark.cpp]: It looks perfect :+1:

**Comment by okdshin** [comment]
LGTM. Thank you for PR!

**Pull request merged**